tests/unit/configuration_subsystem/conftest.py

```diff
@@ -42,7 +42,7 @@ def _generate_config(params=None, setting_file_name=None, initial=True) -> Gener
 
     if setting_file_name:
         settings_file_path = os.path.join(TEST_FIXTURE_DIR, setting_file_name)
-        with open(settings_file_path) as file:
+        with open(file=settings_file_path, encoding="utf-8") as file:
```

Comment:
Why was this necessary? It's the default. Also, it's best to follow what the `tmp_path` does and operate on the pathlib objects. They also have `pathlib.Path.open()` if that's really necessary.

Reply:
@webknjaz see above

Reply:
Oh, it's a separate question but anyway it's not the right place to solve it. Resolved.